src/test/java/org/minions/devfund/benjamin/MovieDatabaseTest.java

```diff
@@ -0,0 +1,114 @@
+        MovieDatabase movieDatabase = new MovieDatabase();
+        movieDatabase.addMovie(NEW_MOVIE, new String[]{ACTOR_UNO, ACTOR_DOS});
+        assertEquals(1, movieDatabase.getMovieList().size());
+        assertEquals(NEW_MOVIE, movieDatabase.getMovieList().get(0).getName());
+        assertEquals(2, movieDatabase.getActorList().size());
+    }
+
+    /**
+     * Test add movie that already exist.
+     */
+    @Test
+    public void testAddMovieAlreadyExist() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        movieDatabase.addMovie(NEW_MOVIE, new String[]{ACTOR_UNO, ACTOR_DOS});
+        movieDatabase.addMovie(NEW_MOVIE, new String[]{ACTOR_UNO, ACTOR_DOS});
+        assertEquals(1, movieDatabase.getMovieList().size());
+        assertEquals(NEW_MOVIE, movieDatabase.getMovieList().get(0).getName());
+        assertEquals(2, movieDatabase.getActorList().size());
+    }
+
+    /**
+     * Test add rating.
+     */
+    @Test
+    public void testAddRatingNewMovie() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        movieDatabase.addMovie(NEW_MOVIE, new String[]{ACTOR_UNO, ACTOR_DOS});
+        final double ten = 10.0;
+        movieDatabase.addRating(NEW_MOVIE, ten);
+        assertEquals(ten, movieDatabase.getMovieList().get(0).getRating(), 0.0);
+    }
+
+    /**
+     * Test update rating.
+     */
+    @Test
+    public void testUpdateRating() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        movieDatabase.addMovie(NEW_MOVIE, new String[]{ACTOR_UNO, ACTOR_DOS});
+        final double ten = 10.0;
+        final double fifteen = 15.0;
+        movieDatabase.addRating(NEW_MOVIE, ten);
+        movieDatabase.updateRating(NEW_MOVIE, fifteen);
+        assertEquals(fifteen, movieDatabase.getMovieList().get(0).getRating(), 0.0);
+    }
+
+    /**
+     * Test get best actor.
+     */
+    @Test
+    public void testGetBestActor() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        movieDatabase.addMovie(NEW_MOVIE, new String[]{ACTOR_UNO, ACTOR_DOS});
+        movieDatabase.addMovie(NEW_MOVIE_TWO, new String[]{ACTOR_UNO});
+        final double ten = 10.0;
+        final double fifteen = 15.0;
+        movieDatabase.addRating(NEW_MOVIE, ten);
+        movieDatabase.addRating(NEW_MOVIE_TWO, fifteen);
+        assertEquals("Crhistian Bale", movieDatabase.getBestActor());
+    }
+
+    /**
+     * Test get best movie.
+     */
+    @Test
+    public void testGetBestMovie() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        movieDatabase.addMovie(NEW_MOVIE, new String[]{ACTOR_UNO, ACTOR_DOS});
+        movieDatabase.addMovie(NEW_MOVIE_TWO, new String[]{ACTOR_UNO});
+        final double ten = 10.0;
+        final double fifteen = 15.0;
+        movieDatabase.addRating(NEW_MOVIE, ten);
+        movieDatabase.addRating(NEW_MOVIE_TWO, fifteen);
+        assertEquals("NewMovie2", movieDatabase.getBestMovie());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use already-defined constant 'NEW_MOVIE_TWO' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)